qfdmd/models.py

```diff
@@ -39,14 +55,87 @@ def __str__(self):
     def sous_categorie_with_carte_display(self):
         return self.sous_categories.filter(afficher_carte=True).first()
 
+    def get_etats_descriptions(self) -> tuple[str, str] | None:
+        # TODO: rename this method
+        # Une fois que les fiches déchet auront toutes
+        # la notion de bon etat / mauvais état stockée en
+        # base de donnée au bon endroit, cette méthode deviendra caduque.
+        text = self.qu_est_ce_que_j_en_fais
+        if "En bon état" not in text and "En mauvais état" not in text:
+            return
+
+        _, _, mauvais_etat_and_rest = text.partition("<b>En bon état</b>")
+        mauvais_etat, _, bon_etat = mauvais_etat_and_rest.partition(
+            "<b>En mauvais état</b>"
+        )
```

Comment:
@kolok

Reply:
peut-être dans la prochaine itération, à toi de voir

Reply:
Alors j'ai fait ça un peu sciemment : le résultat est convenable, mais dégradé. 
Je ne veux pas maintenir un pansement sur une jambe de bois, donc laisser les champs non contribués permet de s'assurer que quelqu'un est repassé derrière. 
Si on l'automatise, j'ai peur qu'on doive maintenir un simili pipeline d'ingestion alors qu'on parle d'une vingtaine de fiches aujourd'hui